.ci/azure-pipelines.yml

```diff
@@ -276,3 +276,6 @@ jobs:
       Ans_39_4:
         python.version: '3.9'
         tox.env: py39-mode_ansible-ansible4
+      Ans_39_5:
+        python.version: '3.9'
+        tox.env: py39-mode_ansible-ansible5
```

Comment:
I'd also like Loc_39_5 & Van_39_5 jobs for Mac1015 & MacLatest. I can do them as follow on PR, or you can include them if you feel like it. I'm happy to merge this either way.

Reply:
On closer inspection there are wider problems with macOS Ci, so disregard this.